kornia/utils/helpers.py

```diff
@@ -29,6 +29,22 @@ def get_cuda_device_if_available(index: int = 0) -> torch.device:
     return dev
 
 
+def get_mps_device_if_available() -> torch.device:
+    """Tries to get mps device, if fail, returns cpu.
+
+    Returns:
+        torch.device
+    """
+    try:
+        if torch.backends.mps.is_available():
+            dev = torch.device('mps')
+        else:
+            dev = torch.device('cpu')
+    except BaseException as e:  # noqa: F841
+        dev = torch.device('cpu')
+    return dev
```

Comment:
i think we can do this way, to avoid the try/catch
```suggestion
    dev = 'cpu'
    if hasattr(torch.backends, 'mps'):
        if torch.backends.mps.is_available():
            dev ='mps'
    return torch.device(dev) 
```
```